app/templates/index.html

```diff
@@ -41,5 +41,33 @@
         </div>
         <h1>{{ title }}</h1>
     </div>
+
+    <h1><u>About Us</u></h1>
```

Comment:
I propose to encapsulate the whole section in a div, so we have each section separately, in this way, you can give it a specific format and decorate it without having to affect the global styles 💡

Reply:
I didn't realize you could do that! I'll get on that right away 👍